meta/canon/canon.go

```diff
@@ -3,47 +3,78 @@ package canon
 
 //go:generate msgp
 
+var (
+	strContinuousDriveDist = []uint{0, 6, 16, 21, 47, 62, 78, 91, 98, 105, 119, 137}
+	strCanonFocusModeDist  = []uint{0, 11, 22, 33, 45, 51, 61, 73}
```

Comment:
Would it be possible to use const's for these values?

Reply:
Yes. Definitely, Thank you.